samples/bluetooth/mesh/light_ctrl/README.rst

```diff
@@ -171,7 +194,7 @@ Configure the Generic OnOff Server model on each element on the :guilabel:`Mesh
 You should now see the following actions:
 
 1. The LED fades from 0% to 100% over a defined transition time following the message :guilabel:`Standby -> On`.
-#. The LED stays at 100% for three seconds :guilabel:`On`.
+#. The LED stays at 100% for six seconds :guilabel:`On`.
```

Comment:
I chose to change this to six, because when recalling the scene specified in toggle_scene.py (with five seconds transition time), the light will begin to fade before reaching max level. I guess this is due to the LC state machine being turned on at the begging of the transition, then taking over before the transition completes.